dataPipelines/gc_thumbnails/utils.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+import fitz
+
+class ThumbnailsCreator:
+    def __init__(
+        self,
+        file_name,
+        output_directory
+    ):
+        self.file_name = file_name
+        self.output_directory = output_directory
+
+    def generate_png(self):
+        p = Path(self.output_directory)
+        if not p.exists():
+            p.mkdir()
+
+        filename = str(self.file_name).split("/")[-1]
+        doc = fitz.open(self.file_name)
+        page = doc.loadPage(0)  # number of page
+        pix = page.getPixmap()
+        out_dir = str(self.output_directory)
```

Comment:
if using string paths, use `output = os.path.join(os.path.abspath(self.output_directory), os.path.splitext(filename)[0] + '.png')`. If using pathlib.Path (for all of those  `self.<x>` vars), use ` output = Path(self.output_directory, self.file_name.with_suffix('.png').name`.